cli/node/daemon_test.go

```diff
@@ -18,7 +19,7 @@ import (
 func TestSocketClient_Send(t *testing.T) {
```

Comment:
split into small tests? :)

Reply:
I need to write an example for this package, so I'll do that at the same time.